Library/Homebrew/dev-cmd/cat.rb

```diff
@@ -44,6 +44,19 @@ def cat
       "cat"
     end
 
+    args.named.to_paths.each do |path|
+      next path if path.exist?
+      path = path.basename(".rb") if args.cask?
+
+      ofail "#{path}'s source doesn't exist on disk."
+    end.presence
+
+    if Homebrew.failed?
+      puts "Either you have made a typo or some taps are not tapped."
+      puts "This is expected behaviour for non-Homebrew developers."
```

Comment:
I suggest dropping the last line entirely, as it's easily misread (one extreme: "us Homebrew maintainers really were expecting too much from you _unwashed masses_").